pkg/image/qemu.go

```diff
@@ -65,7 +65,7 @@ func (o *qemuOperations) ConvertQcow2ToRaw(src, dest string) error {
 func (o *qemuOperations) ConvertQcow2ToRawStream(url *url.URL, dest string) error {
 	jsonArg := fmt.Sprintf("json: {\"file.driver\": \"%s\", \"file.url\": \"%s\", \"file.timeout\": %d}", url.Scheme, url, networkTimeoutSecs)
 
-	_, err := qemuExecFunction(qemuLimits, "qemu-img", "convert", "-f", "qcow2", "-O", "raw", jsonArg, dest)
+	_, err := qemuExecFunction(qemuLimits, "qemu-img", "convert", "-p", "-f", "qcow2", "-O", "raw", jsonArg, dest)
```

Comment:
Should we add progress to the other `qemu-img convert` invocation in this file on line 57?

Reply:
Yes we should.